packages/components/src/FilterBar/context/reducer/clearActiveFilters.ts

```diff
@@ -0,0 +1,16 @@
+import { FiltersState, FiltersValues } from "../types"
+
+export const clearActiveFilters = <ValuesMap extends FiltersValues>(
+  state: FiltersState<ValuesMap>
+): FiltersState<ValuesMap> => {
+  state.activeFilterIds.forEach((id, _, ids) => {
```

Comment:
I do normally prefer writing things in an immutable way, but this seemed straight-forward enough. I'll update it if we ever make our linting yell at anything mutable 😄

Reply:
Sometimes `foreach` is more performant too